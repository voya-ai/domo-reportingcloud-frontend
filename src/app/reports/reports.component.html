<app-report-list></app-report-list>
<app-report-generation-list></app-report-generation-list>
