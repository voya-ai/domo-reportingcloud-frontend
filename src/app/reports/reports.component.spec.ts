import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { MockComponent } from 'src/testing/component-stubs';
import { ReportsComponent } from './reports.component';

describe('ReportsComponent', () => {
  let component: ReportsComponent;
  let fixture: ComponentFixture<ReportsComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [
        ReportsComponent,
        MockComponent({ selector: 'app-report-list' }),
        MockComponent({ selector: 'app-report-generation-list' })
      ]
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(ReportsComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should match the snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
