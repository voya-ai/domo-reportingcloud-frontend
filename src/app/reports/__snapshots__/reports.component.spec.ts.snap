// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReportsComponent should match the snapshot 1`] = `
<app-reports>
  <app-report-list />
  <app-report-generation-list />
  
</app-reports>
`;
